Cargo.toml

```
@@ -22,6 +22,7 @@ fastformat-datatypes = { path = "libraries/datatypes" }
 fastformat-converter = { path = "libraries/converter" }
 fastformat = { path = "libraries/fastformat" }
 
+kornia = { git = "https://github.com/kornia/kornia-rs", tag = "v0.1.6-rc2" }
```

```suggestion
kornia = { git = "https://github.com/kornia/kornia-rs", tag = "v0.1.6-rc3" }
```
just pushed the most recent things :)